README.md

```diff
@@ -202,47 +202,28 @@ If you are interested please see the [contact info on my github page](https://gi
 
 ## Self-Hosting - Docker
 
-Grab the current release number from the [release page](https://github.com/rssnyder/discord-stock-ticker/releases) and expose your designated API port:
+We will go over self-hosting with Docker using [docker-compose](https://docs.docker.com/compose/) only, as this is the generally recommended way to use Docker.
 
-```shell
-docker run -p "8080:8080" ghcr.io/rssnyder/discord-stock-ticker:3.4.1
-```
-
-You can set the config via ENV vars, since we use [namsral/flag](https://github.com/namsral/flag) the variables are the same as the flag inputs, but all uppercase:
-
-```shell
-export ADDRESS="localhost:8080" # address:port to bind http server to.
-export CACHE=false # enable cache for coingecko
-export DB="" # file to store tickers in
-export FREQUENCY=60 # set frequency for all tickers
-export LOGLEVEL=0 # defines the log level. 0=production builds. 1=dev builds.
-export REDISADDRESS="localhost:6379" # address:port for redis server.
-export REDISDB=0 # redis db to use
-export REDISPASSWORD="" # redis password
-```
-
-```shell
-docker run -p "8080:8080" --env CACHE=true ghcr.io/rssnyder/discord-stock-ticker:3.4.1
-```
-
-Then you can pass a volume to store the state (and at the same time, upgrade to using [docker-compose](https://docs.docker.com/compose/)):
+1) Find the current release number from the [release page](https://github.com/rssnyder/discord-stock-ticker/releases).
+2) Create your docker-compose file. The following example is a bare-bones example to get you up and running (anything after the # is a comment and unneeded):
 
 ```shell
 ---
 version: "3"
 services:
 
-  discordstockticker:
-    image: ghcr.io/rssnyder/discord-stock-ticker:3.4.1
-    environment:
-      - DB=/dst.db
-      - CACHE=true
-    volumes:
-      - /home/infra/dst.db:/dst.db
-    ports:
-      - "8112:8080"
+ discordstockticker:
+  image: ghcr.io/rssnyder/discord-stock-ticker:3.9.3
+  environment:
+    -CACHE=false  # Set to true if your bot:coin ratio is going to be greater than 1
```

Comment:
```suggestion
```
lets remove this, almost no one should ever need a cache

Reply:
Probably best. Just wasn't sure if you wanted to keep that in there or not.